### PR TITLE
Apply parallelism

### PR DESCRIPTION
**Pull request by matheusccouto**
I used the module concurrent.futures to apply parallelism to the calculate_population_fitness method.

It solves the issue #15.

### CODE REVIEW

**Comment by lsqshr** [comment]
This branch works. @remiomosowon would you pls consider merging it?

@matheusccouto I would suggest to remove the .gitignore changes here since they are not general

to ignore those directories in your local environment, try adding these lines to .git/info/exclude

File: pyeasyga/pyeasyga.py [hunk 1]
```diff
@@ -135,13 +136,34 @@ def create_initial_population(self):
...
+
+            if "process" in parallel_type.lower():
+                executor = futures.ProcessPoolExecutor(max_workers=n_workers)
+            else:
+                executor = futures.ThreadPoolExecutor(max_workers=n_workers)
+
+            # Create two lists from the same size to be passed as args to the
+            # map function.
+            genes = [individual.genes for individual in self.current_generation]
+            data = [self.seed_data for _ in self.current_generation]
```
Comment (remiomosowon) [inline pyeasyga/pyeasyga.py:160]: `data = [self.seed_data] * len(self.current_generation)` should be noticeably faster given large population and seed_data sizes.

**Comment by remiomosowon** [comment]
Thanks @matheusccouto for the commit and @lsqshr for bringing it to my attention. Merged!

**Comment by rodrick10** [comment]
Thanks @matheusccouto!

How much did your computational time improved after this modification? @lsqshr

**Pull request merged**